openml/flows/flow.py

```diff
@@ -165,8 +166,8 @@ def _to_dict(self) -> dict:
             Flow represented as OrderedDict.
 
         """
-        flow_container = OrderedDict()
-        flow_dict = OrderedDict([('@xmlns:oml', 'http://openml.org/openml')])
+        flow_container = OrderedDict()  # type: 'OrderedDict[str, OrderedDict]'
+        flow_dict = OrderedDict([('@xmlns:oml', 'http://openml.org/openml')])  # type: 'OrderedDict[str, Union[List, str]]'  # noqa E501
```

Comment:
I am a fan of type hints for functions. I am not sure if I am a fan of providing (in-line) type hints for variables. I think it should generally be preferred to remove most ambiguity through documentation and variable names.

Reply:
I only used inline type annotations where mypy couldn't guess the type itself.

Reply:
We discussed offline to keep them for now.